test/TestRefreshRate.c

```diff
@@ -0,0 +1,461 @@
+/**
+ * @file
+ * @brief Unit tests for reading data with virtual i2c communication from sensor
+ * @internal
+ *
+ * @copyright (C) 2017 Melexis N.V.
```

Comment:
I assume you want to put current year here and your name... Otherwise just dump the whole line

```suggestion
```